scrapy_autoextract/cache.py

```diff
@@ -88,3 +89,28 @@ def __setitem__(self, fingerprint: str, value) -> None:
 
     def close(self):
         self.db.close()
+
+
+class ScrapyCloudCollectionCache(_Cache):
+    def __init__(self, project, collection):
+        sc = ScrapinghubClient()
+        self.collection = sc.get_project(project).collections.get_store(collection)
+
+    @classmethod
+    def fingerprint(cls, request: Request) -> str:
+        return request.url
```

Comment:
hey! What's the reason just an URL is used as a fingerprint?

Reply:
Agreed, that probably wasn't optimal. Introduced the same fingerprint mechanism as in `AutoExtractCache`.
It was like this for simplicity - those cached items are later retrieved in QA tool, MATT, and compared with job items. Now I just have to modify the retrieval in MATT to reach to item contents instead of the collection key.